Resources/Prototypes/_RMC14/Roles/Jobs/Survivor/corporate_survivor.yml

```
@@ -35,6 +35,7 @@
     - type: RMCSurvivor
     - type: MotionDetectorTracked
     - type: NpcFactionMember
+    - type: RMCRibbon
       factions:
       - WeYa
```

The ordering here is wrong, 

You can think of this as one configuration and its options.
```yml
    - type: NpcFactionMember
      factions:
      - WeYa
```

You want to add `- type: RMCRibbon` after the factions list.
```suggestion
    - type: NpcFactionMember
      factions:
      - WeYa
    - type: RMCRibbon
```

---

The same should be applied on the other similar files.